frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -142,7 +142,8 @@ export const dashboardLogic = kea<dashboardLogicType>({
                         return dashboard
                     } catch (error: any) {
                         if (error.status === 404) {
-                            return null
+                            lemonToast.error('Dashboard not found')
```

Comment:
If we're showing a proper 404 page, it's redundant to show a toast, right? Usually toasts are more for "thing X happened" messaging rather than "the state of things is Y"

Reply:
Hmmm, I copied Insights but you're right it's a bit redundant on both... 🤔

Reply:
I think there was some talk of removing that from insights at some point, but we never got around to it 😄

Reply:
#12706